src/asciidoc/reference/packages/withkojiDatabase.adoc

```diff
@@ -31,6 +31,30 @@ cd backend
 npm install --save @withkoji/database
 ----
 
+== DatabaseAdapter.valueTypes
```

Comment:
`DatabaseAdapter` in this case is the name used by the developer when they import the Database package. Because the rest of the docs use `import Database from '@withkoji/Database';`, the correct way to access this is by using `Database.valueTypes.increment(1)`

Reply:
Thanks for the clarification! I made a change. Let me know if I'm still missing or misstating something. I'm not so familiar with this pattern :)